component/web/security/src/main/java/org/exoplatform/web/security/codec/CodecInitializer.java

```diff
@@ -135,7 +135,14 @@ protected AbstractCodec initCodec() throws TokenServiceInitializationException {
                 }
             }
 
-            File f = new File(gtnConfDir + "/codec/codeckey.txt");
+            String codecFilePath = PropertyManager.getProperty("exo.security.rememberme.codec.file");
```

Comment:
I think we should remove "rememberme" from the name of the property since it is not dedicated to rememberme feature.